src/Sentry/Laravel/Integration.php

```diff
@@ -5,23 +5,22 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\LazyLoadingViolationException;
 use Illuminate\Routing\Route;
+use function Sentry\addBreadcrumb;
```

Comment:
This is weird, not sure why it happens. Should still be below the regular use statements.